src/Components/SmartComponents/SystemsPage/EdgeDevicesWarning.js

```diff
@@ -2,14 +2,13 @@ import React, { useContext } from 'react';
 import { useIntl } from 'react-intl';
 import { Alert } from '@patternfly/react-core';
 import messages from '../../../Messages';
-import { AccountStatContext } from '../../../Utilities/VulnerabilityRoutes';
-import useFeatureFlag from '../../../Utilities/useFeatureFlag';
+import { AccountStatsContext } from '../../../Utilities/VulnerabilityRoutes';
 
 const EdgeDevicesWarning = () => {
     const intl = useIntl();
-    const { hasEdgeDevices } = useContext(AccountStatContext);
-    const isEdgeParityEnabled = useFeatureFlag('vulnerability.edge_parity');
-    return isEdgeParityEnabled && hasEdgeDevices
+    const { hasEdgeDevices } = useContext(AccountStatsContext);
```

Comment:
I might miss something but it doesn't work for me correctly. I have account that has 0 edge systems but I still see alert (Same issue already exists on stage, but I'm more curious if removal FF will do something wrong)

![Screenshot 2025-04-30 at 13 35 18](https://github.com/user-attachments/assets/5ed41e3c-872b-4486-b55b-49dd3954faac)

you can see 0 edge systems
![Screenshot 2025-04-30 at 13 37 15](https://github.com/user-attachments/assets/e4db92f0-898f-49fd-9b8f-55a59f3c17f2)

Reply:
Very interesting -- this did not work even before the PR -- I will raise this issue in a Slack channel